### PR TITLE
#42 Adds matchOptions to ts types

### LINKED ISSUE #42: TypeScript: `matchOptions` missing from `BreadcrumbsRoute`
Hi!

Thanks for your work on this HoC and for including TS-support! 

I notice the docs and code open for giving `matchOptions` to each breadcrumb route. However, `matchOptions` is missing from the type definitions, so if I try to assign the correct type to the options given to `withBreadcrumbs` I get a type error.

Example: 

```typescript
const breadcrumbs: BreadcrumbsRoute[] = [
  {
    path: '/path/',
    breadcrumb: 'New',
    matchOptions: {
      exact: true,
    },
  },
  {
    path: '/path/:id?',
    breadcrumb: 'Edit',
  },
];
```

The type checker complains that `matchOptions` is not defined. A workaround is to remove the `BreadcrumbsRoute[]` typing, but it would be nice to avoid that.

I can open up a PR that adds the missing definitions if you'd like.

### PR DESCRIPTION
**Pull request by icd2k3**
Fixes #42

### CODE REVIEW

File: types/react-router-breadcrumbs-hoc/index.d.ts [hunk 1]
```diff
@@ -13,9 +13,15 @@ export interface Options {
   pathSection?: string;
 }
 
+export interface MatchOptions {
+  exact?: boolean;
+  strict?: boolean;
```
Comment (wkillerud) [inline types/react-router-breadcrumbs-hoc/index.d.ts]: Thanks for fixing this! 🙌 

From what I can tell [`sensitive?: boolean;`](https://github.com/ReactTraining/react-router/blob/d28d46dce08a5756a085f7e5eebb5169ea59e40b/packages/react-router/modules/matchPath.js#L31) should also be added here for completeness.

Comment (icd2k3) [reply types/react-router-breadcrumbs-hoc/index.d.ts]: nps @wkillerud!

Weird that they haven't included that param in their official [docs](https://reacttraining.com/react-router/web/api/matchPath) yet, but I will include it here for completeness.

Comment (icd2k3) [reply types/react-router-breadcrumbs-hoc/index.d.ts]: Done!

**Comment by icd2k3** [comment]
This is now published as `2.1.5`

**Pull request merged**